content/challenges/medium/9 - Palindrome/func.js

```diff
@@ -9,5 +9,9 @@
  * @returns {boolean}
  */
 module.exports = (str) => {
-    // Your code here
+    let backwards = ''
+    for (i = str.length - 1; i >= 0; i--) {
+        backwards += str[i]
+    }
+    return str === backwards
```

Comment:
so there is something unique about a palindrome that makes the way you did this actually the inefficient way to do it.

If you are given a word with `n` characters you are processing all the characters once so your Big O notation for this is O(n). You can do this in O(n/2). See if you can figure out how